src/codeu/chat/client/commandline/Chat.java

```diff
@@ -34,13 +34,15 @@
 import codeu.chat.common.InterestSet;
```

Comment:
:+1:  https://stackoverflow.com/questions/737673/how-to-read-the-mode-field-of-git-ls-trees-output/8347325#8347325